dietpi/dietpi-software

```diff
@@ -12376,6 +12406,56 @@ _EOF_
 
 		fi
 
+    software_id=181 # PaperMC
+		if (( ${aSOFTWARE_INSTALL_STATE[$software_id]} == 1 )); then
+
+			Banner_Configuration
+      
+      Create_User -d /opt/papermc papermc # This creates a system user named papermc with /opt/papermc as its home dir.
+      # Bedrock compatibility
+      G_WHIP_BUTTON_OK_TEXT='YES'
+			G_WHIP_BUTTON_CANCEL_TEXT='NO'
+      if G_WHIP_YESNO "Would you like to install the Geyser and Floodgate plugins for compatibility with Bedrock Edition?
+      \n Note that this may be buggy."; then
+          Download_Install 'https://ci.nukkitx.com/job/GeyserMC/job/Geyser/job/master/lastSuccessfulBuild/artifact/bootstrap/spigot/target/Geyser-Spigot.jar' /opt/papermc/plugins/Geyser-Spigot.jar
+          Download_Install 'https://ci.nukkitx.com/job/GeyserMC/job/Floodgate/job/development/lastSuccessfulBuild/artifact/bukkit/target/floodgate-bukkit.jar' /opt/papermc/plugins/floodgate-bukkit.jar
+      fi
+      G_EXEC cd /opt/papermc
+      G_EXEC_OUTPUT=1 G_EXEC java -jar /opt/papermc/paperclip.jar nogui
+      G_CONFIG_INJECT 'eula=' 'eula=true' /opt/papermc/eula.txt
+      G_EXEC chown -R papermc /opt/papermc
+      G_EXEC usermod -a -G papermc root
+      G_EXEC chmod -R 760 /opt/papermc
```

Comment:
There is no point in doing the above last two commands:
- `usermod -a -G papermc root` This adds the `root` user to the `papermc` group, but the root user has always full permissions, hence adding it to any group has no effect 😉.
- `chmod -R 760 /opt/papermc` this would then allow the root user write and read (but no execute) permissions, but even with `700` mode the root user can always do everything.

Do only: `G_EXEC chown -R papermc:papermc /opt/papermc`, so the papermc user has full permissions and, aside of root, no other user, which should be perfectly fine.